lib/cocoapods/generator/copy_xcframework_script.rb

```diff
@@ -171,15 +144,16 @@ def script
     return
   fi
   local source="$basepath/$target_path"
-
-  local destination="#{Pod::Target::BuildSettings::XCFRAMEWORKS_BUILD_DIR_VARIABLE}/${name}"
+  local files=( ${source}/*.framework )
```

Comment:
this is the actual fix. we now. tell `rsync` to copy the `path/to/my.framework` into `dest/dir/my.framework` now instead.

This ensures the `--delete` option above does not accidentally delete folders (other frameworks) that are not part of the source.

@amorde one question: we always take`files[0]` as you can see, do we expect we can have multiple `.framework` for a given arch? Is that design allowed by `.xcframework`s?

Reply:
note: this also preserves the functionality of `--delete` in case a pod is bumped and the `.framework` has new files or delete some existing ones.

Reply:
actually thank god for examples this breaks a static library integration.